src/main/java/httpserver/ExceptionHandler.java

```diff
@@ -0,0 +1,26 @@
+package httpserver;
+
+import httpserver.http.request.ReadRequestException;
+import httpserver.server.*;
+
+class ExceptionHandler {
+
+    static void handleException(Exception exception) {
+        if (exception instanceof ClientInputOutputException) {
+            ConsoleWriter.println
+                    (Messages.clientInputOutputExceptionMessage() + exception.getMessage());
+        } else if (exception instanceof ClientSocketException) {
+            ConsoleWriter.println
+                    (Messages.clientSocketExceptionMessage() + exception.getMessage());
+        } else if (exception instanceof ClientCloseConnectionException) {
+            ConsoleWriter.println
+                    (Messages.clientCloseConnectionExceptionMessage() + exception.getMessage());
+        } else if (exception instanceof ReadRequestException) {
+            ConsoleWriter.println
+                    (Messages.readRequestExceptionMessage() + exception.getMessage());
+        } else if (exception instanceof ServerSocketException) {
+            ConsoleWriter.println
+                    (Messages.serverSocketExceptionMessage() + exception.getMessage());
+        }
```

Comment:
It could be helpful to have a default condition that handles any other exception type, so that you capture all error states rather than only those you know about in advance.

Reply:
Good point. I'll add this, thank you